<launch>
    <node pkg="scale_mono_vo_ros" name="vo_node" type="vo_node" output="screen">
        <!-- Image topicname to be subscribed -->
        <param name="topicname_image" type="string" value="/kitti_odometry/left/image_raw"/>
        
        <!-- Topicname of the groundtruth -->
        <param name="topicname_gt" type="string" value="/kitti_odometry/groundtruth"/>

        <!-- topicname of the VO pose estimation to be published -->
        <param name="topicname_pose"            type="string" value="/scale_mono_vo/pose"/>
        <param name="topicname_trajectory"      type="string" value="/scale_mono_vo/trajectory"/>
        <param name="topicname_map_points"      type="string" value="/scale_mono_vo/map_points"/>

        <!-- intrinsic parameter yaml file directory -->
        <param name="directory_intrinsic" type="string" value="$(find scale_mono_vo_ros)/config/mono/kitti_04.yaml"/>
    </node>
</launch>